packages/components/src/Components/CullingInfo/__snapshots__/CullingInformation.test.js.snap

```diff
@@ -3,35 +3,8 @@
 exports[`CullingInformation it should render correctly without any date 1`] = `
 <Fragment>
   <Tooltip
-    appendTo={[Function]}
-    aria="describedby"
-    boundary="window"
-    className=""
-    content="System scheduled for inventory removal in -18297 days"
-    distance={15}
-    enableFlip={true}
-    entryDelay={500}
-    exitDelay={500}
-    flipBehavior={
-      Array [
-        "top",
-        "right",
-        "bottom",
-        "left",
-        "top",
-        "right",
-        "bottom",
-      ]
-    }
-    id=""
-    isAppLauncher={false}
-    isContentLeftAligned={false}
-    isVisible={false}
-    maxWidth="18.75rem"
+    content="System scheduled for inventory removal in -18296 days"
```

Comment:
This should be `content="System scheduled for inventory removal in -18297 days"` because of travis in different timezone.

Reply:
@karelhala thanks! I missed it in that snapshot storm :)